src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/BottomNavigationBarSamplePage.xaml

```diff
@@ -8,32 +8,32 @@
 	  mc:Ignorable="d">
 
 	<Grid>
-		<TextBlock Text="{x:Bind BottomNavBar.SelectedItem.Label, Mode=OneWay}"
+		<TextBlock Text="{x:Bind BottomNavBar.SelectedItem.Tag, Mode=OneWay}"
```

Comment:
I think it was intentional to leave some items without labels to show all options.

But why use Tag instead of label? Label is much cleaner, tag should be kept as a last resort hack :)

Reply:
Yes I'm agree with @MatFillion for the use of "Tag"

Reply:
Here the `Tag` is used to store some content for display, not on the bar item itself, but as the content that is associated with the bar item when it is selected:
![nav-view why i used tag](https://user-images.githubusercontent.com/2359550/84443849-77f0a280-ac0e-11ea-928e-d806e03360ff.gif)
> left: before, right: with tag